zip-0308.rst

```diff
@@ -149,7 +149,8 @@ height minus 1, then this could leak information. Therefore, for target
 height N, implementations SHOULD start generating the transactions at around
 height N-5 (provided that block's timestamp is not more than three hours in
 the past). Each migration transaction SHOULD specify an anchor at height N-10
-for each Sprout JoinSplit description.
+for each Sprout JoinSplit description (funds eligible for migration must have
+at least 10 confirmations).
```

Comment:
The anchor requirement is a "SHOULD", while the eligibility requirement is a (non-RFC2119) "must"; this is inconsistent.

It's already a requirement of the protocol that a JoinSplit can only spend funds from before its anchor, therefore the change is technically redundant. If it's desired to clarify this, then I suggest:
```
for each Sprout JoinSplit description (and therefore only notes created before
this anchor are eligible for migration).
```